great-idea/css/index.css

```diff
@@ -367,25 +366,34 @@ ul li {
 
 .all-prod {
   width: 100%;
+  text-align: justify;
   display: flex;
+  flex-wrap: wrap;
   justify-content: space-around;
 }
 
-.products {
+.product-img {
   width: 100%;
-  padding: 20px;
 }
 
-.top-half div,
-.top-half img {
-  width: 45%;
+.prod-row {
+  width: 100%;
   display: flex;
-  flex-wrap: wrap;
 }
 
-.bottom-half div,
-.bottom-half img {
-  width: 45%;
+.individual-prod {
+  margin: 30px;
+  width: 100%;
   display: flex;
-  flex-wrap: wrap;
+  flex-direction: column;
+  padding: 20px;
+  transition: 0.3s;
+  border: 2px solid white;
+}
+
+.individual-prod:hover {
```

Comment:
beautiful use of the hover css property here to highlight. i like the opacity and rounded borders on the hover as well!